AddingIssuesToEpics.md

```diff
@@ -0,0 +1,149 @@
+# Where do Epics come from?
+
+A roadmap committee designee will create the initial epic for a new metatask in Zenhub. 
+Issues with specific tasks will be made by team leads, PMs, or others on the teams actually doing the work.
+All issues that contribute to a given metatask should be connected to that Epic.
+
+# How to add issues to a Zenhub Epic
+
+## Strategy 1 Make the issues in Zenhub
+
+1. Navigate to the [CFDE Roadmap Board Issues View](https://app.zenhub.com/workspaces/cfde1.roadmap1.board1.5f2050f40fc09500139ce760/board?repos=201342713,210915973,221558839,198670837,192595760,271037371,205294622,235423123,185856706,223434921,240065268,286558188)
+1. Click `New issue` on the top right
+1. Choose the repo where you want the issue to appear in the "Create In" dropdown
+1. Give your issue an informative title
+1. Fill in the comment box with specs, requirements and other relevant info
+1. Assign someone to do the work using the `Assignees` on the top right
+1. Choose any appropriate labels in the right side panel
+1. Under "Estimate" in the right side panel, assign the task a relative difficulty (low is easy, high is hard, it's entirely subjective, don't put a ton of thought into it)
+1. Under "Epic" in the right side panel, choose the Epic to assign the issue to. You can search by name using the search box that pops up
+1. Click `Submit new issue`
+      
+## Strategy 2 Make the issues first, in bulk
+
+1. In Github, create new issues in the appropriate repos
+      1. Navigate to https://github.com/nih1.cfde
```

Comment:
Is this a typo? I can't tell if it's a valid link in my browser.

Reply:
oops